src/app/account/__tests__/utils.test.ts

```diff
@@ -0,0 +1,27 @@
+import { describe, expect } from '@jest/globals';
+import { base64encode, convertBigIntToUint8Array, convertUint8ArrayToBigInt, noPad, safeUrl } from '../utils.js';
+
+describe('utils', () => {
+  it('should properly encode Uint8Array to base64', () => {
```

Comment:
describe('base64encode', () => {
  it('simple', () => {
  });
  it('invalid input', () => {
  });

})